addon/components/attach-popover.js

```diff
@@ -110,6 +112,17 @@ export default Component.extend({
     return `ember-attacher-${this.get('animation')} ${this.get('class') || ''} ${showOrHideClass}`;
   }),
 
+  _style: computed('style', '_transitionDuration', function () {
+    const style = this.get('style');
+    const transitionDuration = this.get('_transitionDuration');
+    assert(
```

Comment:
Doesn't ember normally warn when we pass a style that wasn't marked with `htmlSafe`? If so, I think we should follow suit over asserting.